proxy/scripts/inject-deterministic.py

```diff
@@ -179,6 +188,4 @@ def response(self, flow):
                         )
 
 
-def start():
-    ctx.log.info("Load Deterministic JS")
-    return AddDeterministic()
+addons = [AddDeterministic()]
```

Comment:
Have you verified these changes work in both versions of mitmproxy?

Reply:
tested this on both mitm 2 and mitm4 and works fine